.pipelines/templates/validate-version.yml

```diff
@@ -0,0 +1,27 @@
+parameters:
+- name: UPSTREAM_BRANCH
+  type: string
+  default: remotes/origin/master
+steps:
+- task: Bash@3
+  displayName: 'verify version is changed'
+  name: verify
+  inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # exit on error
+    
+        GIT_DIFF_UPSTREAMBRANCH=${{ parameters.UPSTREAM_BRANCH }}
```

Comment:
Worth mentioning what this logic does? Comparing current branch with upstream branch? (master, right?)